Adding an API to determine installed version of the Spark connector (#27709)
Azure/azure-sdk-for-java: sdk/cosmos/azure-cosmos-spark_3-2_2-12/CHANGELOG.md

### 4.8.0-beta.1 (Unreleased)

#### Features Added

> * Added a simple public Api to determine installed version of the Cosmos Spark connector (`CosmosItemsDataSource.version`/`CosmosChangeFeedDataSource.version`).See [PR 27709](https://github.com/Azure/azure-sdk-for-java/pull/27709)

Reviewer: ```suggestion
* Added an API to determine installed version of the Cosmos Spark connector (`CosmosItemsDataSource.version`/`CosmosChangeFeedDataSource.version`).See [PR 27709](https://github.com/Azure/azure-sdk-for-java/pull/27709)
```
